Source/setmaps.cpp

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file setmaps.cpp
+ *
+ * Implementation of functionality the special quest dungions.

```

Comment:
```suggestion
 * Implementation of functionality the special quest dungeons.
```

same change for header file